packages/models-library/src/models_library/frontend_services_catalog.py

```diff
@@ -93,10 +97,10 @@ def _create_parameter(param_type: str) -> ServiceDockerData:
         contact=OM.email,
         inputs={},
         outputs={
-            "out_1": {
+            output_name: {
```

Comment:
Please, keep this as is: The frontend is expecting to write the parameter value in ```out_1```

Reply:
I guess using a ```List``` instead of a ```Dict``` would require quite some changes in the node-ports and also in the frontend